eXo/Sources/Controllers/HomePage/HomePageViewController.swift

```diff
@@ -58,13 +59,21 @@ class HomePageViewController: UIViewController, WKNavigationDelegate {
         /*
         Set the status bar to white color & the navigation bar is always hidden on this screen
         */
+        self.navigationItem.title = NSLocalizedString("OnBoarding.Title.NavigationBar", comment:"")
```

Comment:
`...NavigationBar` doesn't tell what label it is. Change the key to make it explicit what the label is.